lib/liveValidation/operationSearcher.ts

```diff
@@ -279,10 +293,12 @@ export class OperationSearcher {
       requestInfo.query
     );
     if (ret.matches.length === 0 && ret.reason === undefined) {
-      ret.reason = new LiveValidationError(
-        ErrorCodes.OperationNotFoundInCache.name,
-        `Could not find best match operation for verb "${requestInfo.requestMethod}" for api-version "${requestInfo.apiVersion}" and provider "${requestInfo.providerNamespace}" in the cache.`
-      );
+      meta = getOavErrorMeta(ErrorCodes.OperationNotFoundInCache.name as any, {
```

Comment:
is it a old function , why I can't find its definition ?

Reply:
It's in `errorDefinitions.ts`